e2e/__tests__/index.spec.js

```diff
@@ -0,0 +1,30 @@
+import puppeteer from 'puppeteer'
+import { toMatchImageSnapshot } from 'jest-image-snapshot'
+var browser
+var page
```

Comment:
Why not using `let`?

Reply:
Will change it !!!

_Note a fan of `let` tough_